tests/endtoend/test_durable_functions.py

```diff
@@ -1,9 +1,33 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+from unittest import skipIf
+
+from azure_functions_worker.utils.common import is_envvar_true
 from tests.utils import testutils
+from tests.utils.constants import DEDICATED_DOCKER_TEST, \
+    CONSUMPTION_DOCKER_TEST
 
 
+@skipIf(is_envvar_true(DEDICATED_DOCKER_TEST)
+        or is_envvar_true(CONSUMPTION_DOCKER_TEST),
+        "Docker container cannot set webhook required "
+        "for durable tests")
 class TestDurableFunctions(testutils.WebHostTestCase):
+    env_variables = {}

```

Comment:
Do we need to instantiate env_variables - if we can declare it within `WebHostTestCase` and use it using methods?

Reply:
Fixed